### ISSUE TITLE
OSX Install

### ISSUE THREAD

**Issue by krampstudio** [vim-dist/webvim#43]
I don't have a Mac. I won't have a Mac. I won't use OSX. 
However if this is your situation (sorry), you can try to install webvim and report the steps you've made

**Comment by SebastienElet** [comment]
@krampstudio it was nice to see you yesterday at dotjs. In my opinion 90-95% of web developers use a mac then it's a must have contribution !

**Comment by krampstudio** [comment]
@SebastienElet you're right, this is why this issue exists, but I'll rely on a benevolent contributor for this.

**Comment by PCreations** [comment]
On OSX Sierra 10.12.2 I just did the following and it's working great : 
```
npm install -g eslint csslint jshint jsonlint handlebars
git clone https://github.com/krampstudio/webvim.git ~/.vim
ln -s ~/.vim/.vimrc ~/.vimrc
ln -s ~/.vim/.tern-project ~/.tern-project
vim
```

**Comment by krampstudio** [comment]
@PCreations that's great! how did you install vim on OSX ? with brew ?

**Comment by PCreations** [comment]
@krampstudio Yep, and I'm using iTerm2 (default terminal doesn't send correct keycodes).

**Comment by helmutgranda** [comment]
So you are pretty much installing some node packages and changing vimrc, easy enough.
